### PR TITLE
Remove duplicate imported file

### PR DESCRIPTION
**Pull request by soniasingla**
### What problem does this PR solve?
Remove duplicate imported file

### CODE REVIEW

**Comment by STRRL** [comment]
Hi @soniasingla, as you deleted the `cm`, you should rename the references with `cm` to `common`.

File: pkg/chaosctl/cmd/debug.go [hunk 1]
```diff
@@ -26,7 +26,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/chaos-mesh/chaos-mesh/pkg/chaosctl/common"
```
Comment (soniasingla) [inline pkg/chaosctl/cmd/debug.go:28]: In line 28, it has been declared twice, so removing `cm` should not impact, can you explain me little bit? :)

Comment (STRRL) [reply pkg/chaosctl/cmd/debug.go:28]: for example:

https://github.com/chaos-mesh/chaos-mesh/blob/7e12dea08017445d11f95e5540df5a34d4916ea1/pkg/chaosctl/cmd/debug.go#L76

should be 


```
	clientset, err := common.InitClientSet()
```

**Review by STRRL — CHANGES_REQUESTED** [review]
please rename the import alias `cm` to `common`

**Comment by soniasingla** [comment]
Thanks @STRRL for the review, i have updated the references :)

**Review by AsterNighT — APPROVED** [review]
LGTM

**Review by STRRL — APPROVED** [review]
LGTM

**Pull request merged**